src/docs/markdown/conventions.md

```diff
@@ -42,10 +43,12 @@ The address part may be any of these forms:
 - `/path/to/unix/socket`
 - `/path/to/unix/socket|0200`
 
-The host may be any hostname, resolvable domain name, or IP address.
+The host may be any hostname, resolvable domain name, IP address, or file descriptor number.
 
 In the case of IPv6 addresses, the address must be enclosed in square brackets `[]`. The zone identifier (starting with `%`) is optional (often used for link-local addresses).
 
+In the case of file descriptors, the host must be an unsigned [integer literal](https://go.dev/ref/spec#Integer_literals).
```

Comment:
Could we explain here a bit how this works? Like what does the number actually represent? I don't have a good understanding of how file descriptors are useful, and I assume most users wouldn't understand the point of it either.

In all these docs, I only see examples with `{env.*}` but no actual concrete values, so as a user I would have no idea what the env var values should even look like.